client/transactions/uncaptured/index.tsx

```diff
@@ -0,0 +1,243 @@
+			screenReaderLabel: __( 'Country', 'woocommerce-payments' ),
+			visible: false,
+			isLeftAligned: true,
+		},
+	].filter( Boolean ) as Column[]; // We explicitly define the type because TypeScript can't infer the type post-filtering.
+
+const getFormatedAmountFromString = ( string: string ) => {
+	return `${ string.substring( 0, string.length - 2 ) }.${ string.substring(
+		string.length - 2
+	) }$`;
+};
+
+export const AuthorizationsList = (): JSX.Element => {
+	const columnsToDisplay = getColumns();
+	const {
+		authorizationsSummary,
+		isLoading: isSummaryLoading,
+	} = useAuthorizationsSummary( getQuery() );
+
+	const { authorizations, isLoading } = useAuthorizations( getQuery() );
+
+	const getRiskColor = ( risk: string ) => {
+		switch ( risk ) {
+			case 'high':
+				return 'crimson';
+			case 'normal':
+				return 'green';
+			case 'elevated':
+				return 'darkorange';
+		}
+	};
+
+	const rows = authorizations.map( ( auth ) => {
+		const stringAmount = String( auth.amount );
+
+		const data = {
+			authorization_id: {
+				value: auth.authorization_id,
+				display: auth.authorization_id,
+			},
+			authorized_on: {
+				value: auth.authorized_on,
+				display: auth.authorized_on,
+			},
+			captured_by: {
+				value: auth.captured_by,
+				display: auth.captured_by,
+			},
+			order: {
+				value: auth.order.number,
+				display: (
+					<a
+						href={ auth.order.url }
+					>{ `#${ auth.order.number } from ${ auth.customer_name }` }</a>
+				),
+			},
+			risk_level: {
+				value: auth.risk_level,
+				display: (
+					<span style={ { color: getRiskColor( auth.risk_level ) } }>
+						{ auth.risk_level.charAt( 0 ).toUpperCase() +
+							auth.risk_level.slice( 1 ) }
+					</span>
+				),
+			},
+			amount: {
+				value: auth.amount,
+				display: getFormatedAmountFromString( stringAmount ),
+			},
+			customer_email: {
+				value: auth.customer_email,
+				display: auth.customer_email,
+			},
+			customer_country: {
+				value: auth.customer_country,
+				display: auth.customer_country,
+			},
+		};
+
+		return columnsToDisplay.map(
+			( { key } ) => data[ key ] || { display: null }
+		);
+	} );
+
+	const summary = [];
+
+	summary.push( {
+		value: String( authorizationsSummary.count ),
+		label: 'authorizations',
+	} );
+
+	summary.push( {
+		value: String( authorizationsSummary.total ),
+		label: 'total',
+	} );
+
+	summary.push( {
+		value: getFormatedAmountFromString(
+			String( authorizationsSummary.totalAmount )
+		),
+		label: 'Pending',
+	} );
+
+	return (
+		<Page>
+			<TableCard
+				className="authorizations-list woocommerce-report-table has-search"
+				title={ __(
+					'Uncaptured transactions',
+					'woocommerce-payments'
+				) }
+				isLoading={ isLoading || isSummaryLoading }
+				rowsPerPage={ 10 }
+				totalRows={ 2 }
+				headers={ columnsToDisplay }
+				rows={ rows }
+				summary={ summary }
+				query={ getQuery() }
+				onQueryChange={ onQueryChange }
+				actions={ [
```

Comment:
The `actions` are not needed at this moment, can you please remove those? There's been some discussion and finally we will follow a different approach. As an interim solution, we will add a 'Capture' cta to each of the rows in the table (this will be tackled in a different issue) and leave the 'bulk capture' feature for a later iteration.

Thanks for your patience :)